doc/source/cluster/kubernetes/k8s-ecosystem/prometheus-grafana.md

```diff
@@ -340,7 +340,7 @@ Refer to [this Grafana document](https://grafana.com/tutorials/run-grafana-behin
   * Click "Import".
   * Click "Upload JSON file".
   * Choose a JSON file.
-    * Case 1: If you are using Ray 2.5.0, you can use [config/grafana/default_grafana_dashboard.json](https://github.com/ray-project/kuberay/blob/master/config/grafana/default_grafana_dashboard.json).
+    * Case 1: If you are using Ray 2.9.0, you can use [the sample config files on GitHub repo](https://github.com/ray-project/kuberay/tree/master/config/grafana) (those files that are named `xxx_grafana_dashboard.json`.
```

Comment:
@angelinalg can say more, but fwiw:

- use full form: "GitHub repo" -> "GitHub repository"
- "on GitHub repo" -> "on GitHub" or "in GitHub repository"
- "(those files that are named ..." -> "These file name pattern is ...", or "The file names have a pattern of ..."
  - avoid using past tense
  - remove the parenthesis, or at least match it properly.

Reply:
Done